DigitalLearningSolutions.Data.Tests/Services/TrackerServiceTests.cs

```diff
@@ -65,7 +65,7 @@ public void ProcessQuery_with_GetObjectiveArray_action_passes_query_params()
         }
 
         [Test]
-        public void ProcessQuery_with_GetObjectiveArray_action_correctly_serialises_contentful_response()
+        public void ProcessQuery_with_valid_action_correctly_serialises_contentful_response()

```

Comment:
I'm not sure about this renaming, why are we taking away the reference to the action that is being used here?

Reply:
This is a test for the serialisation of the response, rather than the success of the specific action, which is tested in `ProcessQuery_with_GetObjectiveArray_action_passes_query_params` and its Cc sibling.

Reply:
OK fine, as long as it's covered.